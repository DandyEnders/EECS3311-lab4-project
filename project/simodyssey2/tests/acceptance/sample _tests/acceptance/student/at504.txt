move(W)
play
pass
pass
move(SW)
move(E)
move(SW)
move(E)
move(NE)
land
move(NW)
land
pass
liftoff
move(NE)
liftoff
move(SW)
move(SE)
move(SW)
move(N)
test(14,11,26,32,31)
move(W)
move(N)
move(SW)
land
move(NW)
play
abort
move(E)
play
move(NE)
move(N)
liftoff
move(NW)
move(NE)
move(S)
move(NE)
move(S)
land
move(NW)
move(N)
move(N)
liftoff
liftoff
pass
land
liftoff
liftoff
move(W)
move(NE)
move(NE)
pass
move(W)
move(E)
play
liftoff
move(S)
move(NE)
move(W)
move(SW)
play
test(25,24,31,21,29)
pass
move(NE)
move(NE)
move(SW)
move(S)
move(E)
move(W)
liftoff
move(W)
move(NE)
move(NW)
pass
move(SE)
move(SW)
move(NE)
move(SE)
move(SW)
move(NE)
move(NW)
play
move(S)
wormhole
land
land
wormhole
liftoff
move(NW)
wormhole
move(E)
move(NW)
abort
move(NW)
wormhole
move(E)
move(E)
test(23,29,19,42,46)
move(E)
