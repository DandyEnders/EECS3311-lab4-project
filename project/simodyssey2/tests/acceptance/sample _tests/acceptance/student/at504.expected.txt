  state:0.0, ok
  Welcome! Try test(3,5,7,15,30)
->move(W)
  state:0.1, error
  Negative on that request:no mission in progress.
->play
  state:1.0, mode:play, ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MY--   ----   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   P---   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BW--   ----   *---   ----   PW--   
->pass
  state:2.0, mode:play, ok
  Movement:
    [3,P]:[1,5,2]->[5,4,1]
    [8,P]:[4,3,1]->[3,2,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    MY--   P---   O---   PW--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    BW--   ----   *---   P---   PW--   
->pass
  state:3.0, mode:play, ok
  Movement:
    [5,M]:[3,1,1]->[3,5,1]
    [8,P]:[3,2,1]->[2,1,1]
    [10,B]:[5,1,1]->[5,2,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E---   *---   P---   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   ----   W---   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   PW--   M---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   M*--   ----   P*--   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -W--   B---   *---   P---   PW--   
->move(SW)
  state:4.0, mode:play, ok
  Movement:
    [0,E]:[1,1,1]->[2,5,2]
    [1,P]:[1,3,1]->[2,4,1]
    [2,P]:[1,5,1]->[2,5,3]
    [4,P]:[2,5,1]->[2,4,2]
    [6,P]:[3,4,1]->[2,3,2]
    [7,M]:[4,2,1]->[5,1,1]
    [10,B]:[5,2,1]->[5,1,3]
      reproduced [12,B] at [5,1,4]
      destroyed [7,M] at [5,1,1]
    [11,P]:[5,5,1]->[4,4,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *---   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   ----   WP--   PP--   -EP-   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   -W--   M---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -WBB   ----   *---   P---   -W--   
->move(E)
  state:5.0, mode:play, ok
  Movement:
    [0,E]:[2,5,2]->[2,1,2]
    [2,P]:[2,5,3]->[1,5,1]
    [3,P]:[5,4,1]->[1,4,1]
    [6,P]:[2,3,2]->[3,2,1]
    [8,P]:[2,1,1]->[1,5,2]
    [12,B]:[5,1,4]->[5,3,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *---   ----   P---   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    -E--   ----   W---   PP--   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   P---   O---   -W--   M---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -WB-   ----   *B--   ----   -W--   
->move(SW)
  state:6.0, mode:play, ok
  Explorer got lost in space - out of fuel at Sector:3:5
  The game has ended. You can start a new game.
  Movement:
    [0,E]:[2,1,2]->[3,5,2]
    [1,P]:[2,4,1]->[1,3,1]
    [3,P]:[1,4,1]->[1,3,2]
    [5,M]:[3,5,1]->[2,1,1]
      reproduced [13,M] at [2,1,2]
    [6,P]:[3,2,1]->[2,1,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    ----   *---   PP--   ----   PP--   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    MMP-   ----   W---   -P--   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -Y--   ----   O---   -W--   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    W---   -*--   ----   P*P-   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -WB-   ----   *B--   ----   -W--   
->move(E)
  state:6.1, error
  Negative on that request:no mission in progress.
->move(NE)
  state:6.2, error
  Negative on that request:no mission in progress.
->land
  state:6.3, error
  Negative on that request:no mission in progress.
->move(NW)
  state:6.4, error
  Negative on that request:no mission in progress.
->land
  state:6.5, error
  Negative on that request:no mission in progress.
->pass
  state:6.6, error
  Negative on that request:no mission in progress.
->liftoff
  state:6.7, error
  Negative on that request:no mission in progress.
->move(NE)
  state:6.8, error
  Negative on that request:no mission in progress.
->liftoff
  state:6.9, error
  Negative on that request:no mission in progress.
->move(SW)
  state:6.10, error
  Negative on that request:no mission in progress.
->move(SE)
  state:6.11, error
  Negative on that request:no mission in progress.
->move(SW)
  state:6.12, error
  Negative on that request:no mission in progress.
->move(N)
  state:6.13, error
  Negative on that request:no mission in progress.
->test(14,11,26,32,31)
  state:6.14, error
  Thresholds should be non-decreasing order.
->move(W)
  state:6.15, error
  Negative on that request:no mission in progress.
->move(N)
  state:6.16, error
  Negative on that request:no mission in progress.
->move(SW)
  state:6.17, error
  Negative on that request:no mission in progress.
->land
  state:6.18, error
  Negative on that request:no mission in progress.
->move(NW)
  state:6.19, error
  Negative on that request:no mission in progress.
->play
  state:7.0, mode:play, ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EW--   P---   PY--   Y---   Y---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    *---   B---   Y---   ----   PW--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    P---   ----   O---   P---   ----   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   PY--   ----   ----   A---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    P---   ----   ----   P*--   *---   
->abort
  state:7.1, ok
  Mission aborted. Try test(3,5,7,15,30)
->move(E)
  state:7.2, error
  Negative on that request:no mission in progress.
->play
  state:8.0, mode:play, ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EW--   P---   ----   ----   PPB-   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   P*--   W---   B*--   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   ----   O---   W---   BY--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    B---   ----   BB--   J*--   BY--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    P---   P---   Y---   BW--   B---   
->move(NE)
  state:9.0, mode:play, ok
  Movement:
    [0,E]:[1,1,1]->[5,2,2]
    [5,P]:[2,1,1]->[2,5,2]
    [7,B]:[2,4,1]->[3,4,2]
    [8,P]:[2,5,1]->[2,4,1]
    [9,B]:[3,5,1]->[3,1,1]
    [12,B]:[4,3,2]->[5,3,2]
    [15,P]:[5,1,1]->[5,2,3]
    [16,P]:[5,2,1]->[4,2,1]
    [17,B]:[5,4,1]->[3,2,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -W--   P---   ----   ----   PPB-   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   P*--   W---   P*--   -P--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    B---   B---   O---   WB--   -Y--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    B---   P---   B---   J*--   BY--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   -EP-   YB--   -W--   B---   
->move(N)
  state:10.0, mode:play, ok
  Movement:
    [0,E]:[5,2,2]->[4,2,2]
    [3,P]:[1,5,2]->[5,5,2]
    [4,B]:[1,5,3]->[5,5,3]
    [7,B]:[3,4,2]->[5,3,3]
      reproduced [19,B] at [5,3,4]
    [14,B]:[4,5,1]->[5,1,1]
    [17,B]:[3,2,1]->[4,1,2]
      reproduced [20,B] at [4,1,3]
    [18,B]:[5,5,1]->[4,4,3]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -W--   P---   ----   ----   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   P*--   W---   P*--   -P--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    B---   ----   O---   W---   -Y--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    BBB-   PE--   B---   J*B-   -Y--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    B---   --P-   YBBB   -W--   -PB-   
->liftoff
  state:10.1, mode:play, error
  Negative on that request:you are not on a planet at Sector:4:2
->move(NW)
  state:11.0, mode:play, ok
  Explorer got lost in space - out of fuel at Sector:3:1
  The game has ended. You can start a new game.
  Movement:
    [0,E]:[4,2,2]->[3,1,2]
    [1,P]:[1,2,1]->[5,1,2]
    [2,P]:[1,5,1]->[1,1,1]
    [3,P]:[5,5,2]->[1,1,3]
    [7,B]:[5,3,3]->[5,2,1]
    [9,B]:[3,1,1]->[2,1,1]
      reproduced [21,B] at [2,1,2]
    [10,B]:[4,1,1]->[4,5,1]
    [11,B]:[4,3,1]->[5,4,1]
    [12,B]:[5,3,2]->[5,2,2]
      reproduced [22,B] at [5,2,4]
    [13,J]:[4,4,1]->[4,3,1]
    [14,B]:[5,1,1]->[5,5,1]
      reproduced [23,B] at [5,5,2]
    [16,P]:[4,2,1]->[3,2,1]
    [17,B]:[4,1,2]->[4,2,1]
    [19,B]:[5,3,4]->[4,4,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    PWP-   ----   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    BB--   P*--   W---   P*--   -P--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   P---   O---   W---   -Y--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    --B-   B---   J---   B*B-   BY--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    -P--   BBPB   Y---   BW--   BBB-   
->move(NE)
  state:11.1, error
  Negative on that request:no mission in progress.
->move(S)
  state:11.2, error
  Negative on that request:no mission in progress.
->move(NE)
  state:11.3, error
  Negative on that request:no mission in progress.
->move(S)
  state:11.4, error
  Negative on that request:no mission in progress.
->land
  state:11.5, error
  Negative on that request:no mission in progress.
->move(NW)
  state:11.6, error
  Negative on that request:no mission in progress.
->move(N)
  state:11.7, error
  Negative on that request:no mission in progress.
->move(N)
  state:11.8, error
  Negative on that request:no mission in progress.
->liftoff
  state:11.9, error
  Negative on that request:no mission in progress.
->liftoff
  state:11.10, error
  Negative on that request:no mission in progress.
->pass
  state:11.11, error
  Negative on that request:no mission in progress.
->land
  state:11.12, error
  Negative on that request:no mission in progress.
->liftoff
  state:11.13, error
  Negative on that request:no mission in progress.
->liftoff
  state:11.14, error
  Negative on that request:no mission in progress.
->move(W)
  state:11.15, error
  Negative on that request:no mission in progress.
->move(NE)
  state:11.16, error
  Negative on that request:no mission in progress.
->move(NE)
  state:11.17, error
  Negative on that request:no mission in progress.
->pass
  state:11.18, error
  Negative on that request:no mission in progress.
->move(W)
  state:11.19, error
  Negative on that request:no mission in progress.
->move(E)
  state:11.20, error
  Negative on that request:no mission in progress.
->play
  state:12.0, mode:play, ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EW--   ----   P---   P*--   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    B---   *---   P---   P---   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    PP--   PW--   O---   Y---   Y---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    BJ--   BY--   Y---   *---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   ----   Y---   ----   ----   
->liftoff
  state:12.1, mode:play, error
  Negative on that request:you are not on a planet at Sector:1:1
->move(S)
  state:13.0, mode:play, ok
  Movement:
    [0,E]:[1,1,1]->[2,1,2]
    [3,B]:[2,1,1]->[1,1,1]
    [4,P]:[2,3,1]->[1,3,2]
    [6,P]:[3,1,1]->[2,2,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    BW--   ----   PP--   P*--   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    -E--   *P--   ----   P---   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    -P--   PW--   O---   Y---   Y---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    BJ--   BY--   Y---   *---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   ----   Y---   ----   ----   
->move(NE)
  state:14.0, mode:play, ok
  Movement:
    [0,E]:[2,1,2]->[1,2,1]
    [3,B]:[1,1,1]->[3,1,1]
      reproduced [12,B] at [3,1,3]
    [4,P]:[1,3,2]->[5,2,1]
    [5,P]:[2,4,1]->[2,5,1]
    [7,P]:[3,1,2]->[4,2,3]
    [9,B]:[4,1,1]->[3,1,2]
    [11,B]:[4,2,1]->[4,3,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -W--   E---   P---   P*--   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *P--   ----   ----   P---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    BBB-   PW--   O---   Y---   Y---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    -J--   -YP-   YB--   *---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   P---   Y---   ----   ----   
->move(W)
  state:15.0, mode:play, ok
  Explorer got lost in space - out of fuel at Sector:1:1
  The game has ended. You can start a new game.
  Movement:
    [0,E]:[1,2,1]->[1,1,1]
    [1,P]:[1,3,1]->[2,2,3]
    [5,P]:[2,5,1]->[3,5,2]
    [8,P]:[3,2,1]->[3,3,2]
    [10,J]:[4,1,2]->[5,2,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -W--   ----   ----   P*--   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   *PP-   ----   ----   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    BBB-   -W--   O---   Y---   YP--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    ----   -YP-   YB--   *---   ----   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PJ--   Y---   ----   ----   
->move(SW)
  state:15.1, error
  Negative on that request:no mission in progress.
->play
  state:16.0, mode:play, ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    EB--   ----   B---   *---   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   A---   P---   B---   Y---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   BW--   O---   ----   W---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    BW--   Y---   *---   ----   P---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    Y---   PP--   P---   BP*-   JY--   
->test(25,24,31,21,29)
  state:16.1, mode:play, error
  To start a new mission, please abort the current one first.
->pass
  state:17.0, mode:play, ok
  Explorer got destroyed by asteroid (id: 4) at Sector:1:1
  The game has ended. You can start a new game.
  Movement:
    [2,B]:[1,3,1]->[1,4,2]
    [4,A]:[2,2,1]->[1,1,3]
      destroyed [0,E] at [1,1,1]
      destroyed [1,B] at [1,1,2]
    [6,B]:[2,4,1]->[3,4,1]
    [8,B]:[4,1,1]->[3,2,3]
    [9,P]:[4,5,1]->[3,1,1]
    [12,P]:[5,3,1]->[4,2,2]
    [13,B]:[5,4,1]->[4,5,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    --A-   ----   ----   *B--   P---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   P---   ----   Y---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    P---   BWB-   O---   B---   W---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    -W--   YP--   *---   ----   B---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    Y---   PP--   ----   -P*-   JY--   
->move(NE)
  state:17.1, error
  Negative on that request:no mission in progress.
->move(NE)
  state:17.2, error
  Negative on that request:no mission in progress.
->move(SW)
  state:17.3, error
  Negative on that request:no mission in progress.
->move(S)
  state:17.4, error
  Negative on that request:no mission in progress.
->move(E)
  state:17.5, error
  Negative on that request:no mission in progress.
->move(W)
  state:17.6, error
  Negative on that request:no mission in progress.
->liftoff
  state:17.7, error
  Negative on that request:no mission in progress.
->move(W)
  state:17.8, error
  Negative on that request:no mission in progress.
->move(NE)
  state:17.9, error
  Negative on that request:no mission in progress.
->move(NW)
  state:17.10, error
  Negative on that request:no mission in progress.
->pass
  state:17.11, error
  Negative on that request:no mission in progress.
->move(SE)
  state:17.12, error
  Negative on that request:no mission in progress.
->move(SW)
  state:17.13, error
  Negative on that request:no mission in progress.
->move(NE)
  state:17.14, error
  Negative on that request:no mission in progress.
->move(SE)
  state:17.15, error
  Negative on that request:no mission in progress.
->move(SW)
  state:17.16, error
  Negative on that request:no mission in progress.
->move(NE)
  state:17.17, error
  Negative on that request:no mission in progress.
->move(NW)
  state:17.18, error
  Negative on that request:no mission in progress.
->play
  state:18.0, mode:play, ok
  Movement:none
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    E*--   P---   B---   J---   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    P---   ----   ----   PY--   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    JB--   W---   O---   ----   W---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    *---   B*--   ----   W---   W---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PB*-   PW--   P---   ----   
->move(S)
  state:19.0, mode:play, ok
  Movement:
    [0,E]:[1,1,1]->[2,1,2]
    [1,P]:[1,2,1]->[1,3,2]
    [7,B]:[3,1,2]->[3,2,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -*--   ----   BP--   J---   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    PE--   ----   ----   PY--   ----   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    J---   WB--   O---   ----   W---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    *---   B*--   ----   W---   W---   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PB*-   PW--   P---   ----   
->wormhole
  state:19.1, mode:play, error
  Explorer couldn't find wormhole at Sector:2:1
->land
  state:19.2, mode:play, error
  Negative on that request:no yellow dwarf at Sector:2:1
->land
  state:19.3, mode:play, error
  Negative on that request:no yellow dwarf at Sector:2:1
->wormhole
  state:19.4, mode:play, error
  Explorer couldn't find wormhole at Sector:2:1
->liftoff
  state:19.5, mode:play, error
  Negative on that request:you are not on a planet at Sector:2:1
->move(NW)
  state:20.0, mode:play, ok
  Movement:
    [0,E]:[2,1,2]->[1,5,1]
    [2,B]:[1,3,1]->[2,4,3]
    [3,J]:[1,4,1]->[2,5,1]
    [4,P]:[2,1,1]->[2,5,2]
    [6,J]:[3,1,1]->[3,2,3]
    [8,B]:[4,2,1]->[5,2,4]
    [10,B]:[5,2,2]->[1,1,1]
    [11,P]:[5,3,1]->[4,3,1]
    [12,P]:[5,4,1]->[4,5,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    B*--   ----   -P--   ----   E---   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   ----   PYB-   JP--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   WBJ-   O---   ----   W---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    *---   -*--   P---   W---   WP--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   P-*B   -W--   ----   ----   
->wormhole
  state:20.1, mode:play, error
  Explorer couldn't find wormhole at Sector:1:5
->move(E)
  state:21.0, mode:play, ok
  Movement:
    [0,E]:[1,5,1]->[1,1,3]
    [1,P]:[1,3,2]->[5,3,1]
    [2,B]:[2,4,3]->[2,3,1]
      reproduced [13,B] at [2,3,2]
    [7,B]:[3,2,2]->[5,2,2]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    B*E-   ----   ----   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   BB--   PY--   JP--   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   W-J-   O---   ----   W---   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    *---   -*--   P---   W---   WP--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PB*B   PW--   ----   ----   
->move(NW)
  state:22.0, mode:play, ok
  Movement:
    [0,E]:[1,1,3]->[5,5,1]
    [1,P]:[5,3,1]->[4,4,2]
    [2,B]:[2,3,1]->[3,3,2]
    [4,P]:[2,5,2]->[3,5,2]
    [8,B]:[5,2,4]->[1,3,1]
      reproduced [14,B] at [1,3,2]
    [10,B]:[1,1,1]->[5,5,2]
      reproduced [15,B] at [5,5,3]
    [11,P]:[4,3,1]->[4,2,1]
    (1:1)  (1:2)  (1:3)  (1:4)  (1:5)  
    -*--   ----   BB--   ----   ----   
    (2:1)  (2:2)  (2:3)  (2:4)  (2:5)  
    ----   ----   -B--   PY--   J---   
    (3:1)  (3:2)  (3:3)  (3:4)  (3:5)  
    ----   W-J-   O---   ----   WP--   
    (4:1)  (4:2)  (4:3)  (4:4)  (4:5)  
    *---   P*--   ----   WP--   WP--   
    (5:1)  (5:2)  (5:3)  (5:4)  (5:5)  
    ----   PB*-   -W--   ----   EBB-   
->abort
  state:22.1, ok
  Mission aborted. Try test(3,5,7,15,30)
->move(NW)
  state:22.2, error
  Negative on that request:no mission in progress.
->wormhole
  state:22.3, error
  Negative on that request:no mission in progress.
->move(E)
  state:22.4, error
  Negative on that request:no mission in progress.
->move(E)
  state:22.5, error
  Negative on that request:no mission in progress.
->test(23,29,19,42,46)
  state:22.6, error
  Thresholds should be non-decreasing order.
->move(E)
  state:22.7, error
  Negative on that request:no mission in progress.
